plugins/custom-mui-theme/theme/theme.json

```diff
@@ -0,0 +1,218 @@
+    "body2": {
+      "color": "rgba(0, 0, 0, 0.87)",
+      "fontFamily": "\"Roboto\", \"Helvetica\", \"Arial\", sans-serif",
+      "lineHeight": "1.71429em",
+      "fontSize": "0.875rem",
+      "fontWeight": 500
+    },
+    "caption": {
+      "color": "rgba(0, 0, 0, 0.54)",
+      "fontFamily": "\"Roboto\", \"Helvetica\", \"Arial\", sans-serif",
+      "lineHeight": "1.375em",
+      "fontSize": "0.75rem",
+      "fontWeight": 400
+    },
+    "fontSize": 14,
+    "fontWeightMedium": 500,
+    "title": {
+      "color": "rgba(0, 0, 0, 0.87)",
+      "fontFamily": "\"Roboto\", \"Helvetica\", \"Arial\", sans-serif",
+      "lineHeight": "1.16667em",
+      "fontSize": "1.3125rem",
+      "fontWeight": 500
+    },
+    "subheading": {
+      "color": "rgba(0, 0, 0, 0.87)",
+      "fontFamily": "\"Roboto\", \"Helvetica\", \"Arial\", sans-serif",
+      "lineHeight": "1.5em",
+      "fontSize": "1rem",
+      "fontWeight": 400
+    },
+    "body1": {
+      "color": "rgba(0, 0, 0, 0.87)",
+      "fontFamily": "\"Roboto\", \"Helvetica\", \"Arial\", sans-serif",
+      "lineHeight": "1.46429em",
+      "fontSize": "0.875rem",
+      "fontWeight": 400
+    }
+  },
+  "zIndex": {
+    "modal": 1300,
+    "snackbar": 1400,
+    "drawer": 1200,
+    "appBar": 1100,
+    "mobileStepper": 1000,
+    "tooltip": 1500
+  },
+  "shape": { "borderRadius": 4 },
+  "props": {},
+  "spacing": { "unit": 8 },
+  "palette": {
+    "tonalOffset": 0.2,
+    "background": { "paper": "#fff", "default": "#fafafa" },
+    "contrastThreshold": 3,
+    "grey": {
+      "50": "#fafafa",
+      "100": "#f5f5f5",
+      "200": "#eeeeee",
+      "300": "#e0e0e0",
+      "400": "#bdbdbd",
+      "500": "#9e9e9e",
+      "600": "#757575",
+      "700": "#616161",
+      "800": "#424242",
+      "900": "#212121",
+      "A700": "#616161",
+      "A100": "#d5d5d5",
+      "A400": "#303030",
+      "A200": "#aaaaaa"
+    },
+    "text": {
+      "primary": "rgba(0, 0, 0, 0.87)",
+      "secondary": "rgba(0, 0, 0, 0.54)",
+      "disabled": "rgba(0, 0, 0, 0.38)",
+      "hint": "rgba(0, 0, 0, 0.38)"
+    },
+    "divider": "rgba(0, 0, 0, 0.12)",
+    "secondary": {
+      "main": "#FFFF00",
+      "light": "rgb(255, 255, 51)",
+      "dark": "rgb(178, 178, 0)",
+      "contrastText": "rgba(0, 0, 0, 0.87)"
+    },
+    "common": { "black": "#000", "white": "#fff" },
+    "error": {
+      "light": "#e57373",
+      "main": "#f44336",
+      "dark": "#d32f2f",
+      "contrastText": "#fff"
+    },
+    "type": "light",
+    "action": {
+      "hoverOpacity": 0.08,
+      "hover": "rgba(0, 0, 0, 0.08)",
+      "selected": "rgba(0, 0, 0, 0.14)",
+      "disabledBackground": "rgba(0, 0, 0, 0.12)",
+      "disabled": "rgba(0, 0, 0, 0.26)",
+      "active": "rgba(0, 0, 0, 0.54)"
+    },
+    "primary": {
+      "main": "#B71C1C",
+      "light": "rgb(197, 73, 73)",
+      "dark": "rgb(128, 19, 19)",
+      "contrastText": "#fff"
+    }
+  },
+  "themeName": "Thunderbird Yellow Goose"
+}
```

Comment:
comment: and I never use the json file, I would prefer having the theme in js/ts so that we can more dynamic support
can check frontend repo `theme/index.ts` and `theme/colors.tx`